net/Legcord/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        Legcord Legcord 1.0.6 v
+github.tarball_from archive
+distname            ${github.project}-${github.version}
```

Comment:
does the default not work?

Reply:
it didn't the first time I tried it, but just trying it now, it does just fine. Must have been something I was doing. Yanked the 'archive' line

Reply:
You need the “github,tarball_from” line; the “distname” should be redundabt I think

Reply:
well, for some reason it works inconsistently on `...from releases`. so I switched back to archive and yanked the distname line.

sorry for the running around in circles.